### ISSUE TITLE
Images in Django getting clipped/flattened 

### ISSUE THREAD

**Issue by rocky** [mathics/Mathics#1198]
To reproduce, bring up Django and run the gallery. Here is a screenshot of what I get in google.chrome and firefox:

![mathics-clipped-images](https://user-images.githubusercontent.com/8851/112317259-3e12f900-8c82-11eb-83e8-1d29efe7aa1c.png)

I suspect this has to do with some of the Boxes changes since this didn't occur in 2.0.0. @mmatera  ?

**Comment by mmatera** [comment]
I tested the same code in my installation and seems to work without problems. What happens if you add a //FullForm at the end of the Graphics expressions?

**Comment by rocky** [comment]
Here ya go: 

```
BarChart[{{1, 2, 3}, {2, 3, 4}}] // FullForm
=
System`RowBox[System`List["StandardForm", "[", System`RowBox[System`List["FullForm", "[", System`RowBox[System`List["Graphics", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["EdgeForm", "[", System`RowBox[System`List["GrayLevel", "[", "0", "]"]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["0.15999999999999998", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["1.06", ", ", "1."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.9411764705882353", ", ", "0.6470588235294118", ", ", "0.6901960784313725"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["1.12", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["2.02", ", ", "2."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.5490196078431373", ", ", "0.5215686274509804", ", ", "0.21176470588235294"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["2.08", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["2.98", ", ", "3."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.9803921568627451", ", ", "0.8156862745098039", ", ", "0.38823529411764707"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["3.24", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["4.140000000000001", ", ", "2."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.9411764705882353", ", ", "0.6470588235294118", ", ", "0.6901960784313725"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["4.2", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["5.1000000000000005", ", ", "3."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.5490196078431373", ", ", "0.5215686274509804", ", ", "0.21176470588235294"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Style", "[", System`RowBox[System`List[System`RowBox[System`List["Rectangle", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["5.16", ", ", "0"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["6.0600000000000005", ", ", "4."]], "]"]]]], "]"]], ", ", System`RowBox[System`List["RGBColor", "[", System`RowBox[System`List["0.9803921568627451", ", ", "0.8156862745098039", ", ", "0.38823529411764707"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Line", "[", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["0.", ", ", "0."]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["6.0600000000000005", ", ", "0."]], "]"]]]], "]"]], "]"]], ", ", System`RowBox[System`List["FaceForm", "[", System`RowBox[System`List["GrayLevel", "[", "0", "]"]], "]"]], ", ", System`RowBox[System`List["Line", "[", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["0.15999999999999998", ", ", "0."]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["0.15999999999999998", ", ", "-0.2"]], "]"]]]], "]"]], "]"]], ", ", System`RowBox[System`List["Line", "[", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["2.98", ", ", "0."]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["2.98", ", ", "-0.2"]], "]"]]]], "]"]], "]"]], ", ", System`RowBox[System`List["Line", "[", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["3.24", ", ", "0."]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["3.24", ", ", "-0.2"]], "]"]]]], "]"]], "]"]], ", ", System`RowBox[System`List["Line", "[", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["6.0600000000000005", ", ", "0."]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["6.0600000000000005", ", ", "-0.2"]], "]"]]]], "]"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["$OptionSyntax", ", ", "Ignore"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["AspectRatio", ", ", System`RowBox[System`List["Power", "[", System`RowBox[System`List["GoldenRatio", ", ", "-1"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["Axes", ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["False", ", ", "True"]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["AxesStyle", ", ", System`RowBox[System`List["List", "[", "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["Background", ", ", "Automatic"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["ChartLabels", ", ", "None"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["ChartLegends", ", ", "None"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["ChartStyle", ", ", "Automatic"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["ImageSize", ", ", "Automatic"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["LabelStyle", ", ", System`RowBox[System`List["List", "[", "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["Mesh", ", ", "None"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["PlotRange", ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List[System`RowBox[System`List["List", "[", System`RowBox[System`List["-0.1", ", ", "6.0600000000000005"]], "]"]], ", ", System`RowBox[System`List["List", "[", System`RowBox[System`List["0.", ", ", "4."]], "]"]]]], "]"]]]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["PlotRangePadding", ", ", "Automatic"]], "]"]], ", ", System`RowBox[System`List["Rule", "[", System`RowBox[System`List["TicksStyle", ", ", System`RowBox[System`List["List", "[", "]"]]]], "]"]]]], "]"]], "]"]], "]"]]
```

I suspect this is the same thing you get too.

**Comment by rocky** [comment]
For what it is worht I compared with a version that does work, and the FullForm output is exactly the same.

**Comment by mmatera** [comment]
#1210 fixes this, together with [#28 @ mathics-django](https://github.com/Mathics3/mathics-django/pull/28)

**Comment by rocky** [comment]
Yes fixed by #1210. We need though to remember to check the gallery periodically, or better, come with some  automated way to test this.
